Formula/fizz.rb

```diff
@@ -31,13 +31,16 @@ class Fizz < Formula
   depends_on "zstd"
 
   fails_with gcc: "5"
-
   def install
+    inreplace "fizz/CMakeLists.txt",
+    "set(FIZZ_SOURCES\n",
+    "set(FIZZ_SOURCES\n third-party/libsodium-aegis/private/softaes.c"
+
```

Comment:
There must be a saner way of doing this.

Reply:
Weird. Is this code that depends on an unreleased version of `libsodium`, or something?

Reply:
I suggest upstreaming a patch. Seems like they just forgot to add this here: https://github.com/facebookincubator/fizz/commit/ecab8fc4434667387db4556afbe05062636be58e

Reply:
Looks like someone else got there first: https://github.com/facebookincubator/fizz/commit/0dc415e2e7dade586b445946a939d4f8ff15e8d2

Will wait for the next release

Reply:
Cool, we can backport that then. Will push an update.